From eb4df1f5152880519b3600e8dcf72d81ec87ecbf Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
Date: Thu, 29 Aug 2024 13:02:51 +0530
Subject: [PATCH 037/128] riscv: dts: eswin: harmonize device-tree

- use eswin/eic7700-hifive-premier-p550.dtb as device-tree name
- use "sifive,hifive-premier-p550", "eswin,eic7700" as compatible strings

Signed-off-by: Pritesh Patel <pritesh.patel@einfochips.com>
Signed-off-by: Heinrich Schuchardt <heinrich.schuchardt@canonical.com>
---
 arch/riscv/boot/dts/eswin/Makefile                              | 2 +-
 ...{hifive-premier-p550.dts => eic7700-hifive-premier-p550.dts} | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)
 rename arch/riscv/boot/dts/eswin/{hifive-premier-p550.dts => eic7700-hifive-premier-p550.dts} (99%)

diff --git a/arch/riscv/boot/dts/eswin/Makefile b/arch/riscv/boot/dts/eswin/Makefile
index b386475d14f2..f550122f0335 100644
--- a/arch/riscv/boot/dts/eswin/Makefile
+++ b/arch/riscv/boot/dts/eswin/Makefile
@@ -1,3 +1,3 @@
 # SPDX-License-Identifier: GPL-2.0
-dtb-$(CONFIG_SOC_SIFIVE) += hifive-premier-p550.dtb
+dtb-$(CONFIG_SOC_SIFIVE) += eic7700-hifive-premier-p550.dtb
 obj-$(CONFIG_BUILTIN_DTB) += $(addsuffix .o, $(dtb-y))
diff --git a/arch/riscv/boot/dts/eswin/hifive-premier-p550.dts b/arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts
similarity index 99%
rename from arch/riscv/boot/dts/eswin/hifive-premier-p550.dts
rename to arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts
index e2f9c2ce62a2..e6e7ef8cb688 100644
--- a/arch/riscv/boot/dts/eswin/hifive-premier-p550.dts
+++ b/arch/riscv/boot/dts/eswin/eic7700-hifive-premier-p550.dts
@@ -41,7 +41,7 @@ / {
 	#address-cells = <2>;
 	#size-cells = <2>;
 	model = "SiFive HiFive Premier P550";
-	compatible = "sifive,hifive-premier-p550";
+	compatible = "sifive,hifive-premier-p550", "eswin,eic7700";
 
 	aliases {
 		serial0 = &d0_uart0;
-- 
2.47.0

